cumulus_library/studies/discovery/code_detection.py

```diff
@@ -61,22 +48,21 @@ def prepare_queries(self, cursor: object, schema: str, *args, **kwargs):
         code_sources = []
         for code_definition in code_definitions.code_list:
             if any(
-                x not in code_definition.keys() for x in ["table_name", "column_name"]
+                x not in code_definition.keys()
+                for x in ["table_name", "column_hierarchy"]
             ):
```

Comment:
nit:
```
required_keys = {"table_name", "column_hierarchy"}
if not required_keys.issubset(code_definition):
```

or leave it as is - this was just your friendly neighborhood few-liner suggestion